homeassistant/components/sonos/strings.json

```
@@ -53,6 +53,14 @@
     "sensor": {
       "audio_input_format": {
         "name": "Audio input format"
+      },
+      "power_source": {
+        "name": "Power source",
+        "state": {
+          "battery": "Battery",
+          "charging_base": "Charging Base",
```

```suggestion
          "charging_base": "Charging base",
```
Or is this a brand name